tools/dependencies/curl.sh

```diff
@@ -35,6 +35,7 @@ if [[ ! -f $DEPS_PATH/lib/libcurl.a ]]; then
         CONFIG_FLAG="--with-darwinssl"
     fi
     ./configure $CONFIG_FLAG \
+                --without-libidn2 \
```

Comment:
> No, there's no known security reason to avoid enabling libidn2 in curl builds.
For generic curl builds I would recommend building with it so that users can
use international domain names in URLs

https://curl.haxx.se/mail/lib-2017-10/0158.html

Why disable this?